python/ray/dag/tests/experimental/test_accelerated_dag.py

```diff
@@ -136,18 +136,75 @@ def test_basic(ray_start_regular):
     compiled_dag.teardown()
 
 
-def test_multiple_returns_not_supported(ray_start_regular):
+def test_two_returns_first(ray_start_regular):
+    a = Actor.remote(0)
+    with InputNode() as i:
+        o1, o2 = a.return_two.bind(i)
+        dag = o1
+
+    res = dag.execute(1)
+    assert ray.get(res) == 1
+
+    compiled_dag = dag.experimental_compile()
+    res = ray.get(compiled_dag.execute(1))
+    assert res == 1
+
+    compiled_dag.teardown()
+
+
+def test_two_returns_second(ray_start_regular):
```

Comment:
can you add a test that has num_returns=2, but it actually returns 3 entries?

Also if num_returns==1 and if we return 2, is it going to be a single tuple with 1 reference?

Reply:
Done.